drafts/linux-recommendations.md

```diff
@@ -0,0 +1,228 @@
+Meaning that if your main host system is compromised, you should ultimately consider the guess insecure too. 
+
+The most popular choice for virtualization is the free VirtualBox solution https://www.virtualbox.org/
+
+
+### Buy a laptop with Linux pre-installed
+
+Some specialized computer vendors sell privacy-preserving laptops pre-installed with Linux. The two most popular ones being: 
+
+- System76 https://system76.com/ & its Pop OS! distribution
+- Purism Librem https://puri.sm/products/ & its PureOS distribution
+- Tuxedo https://www.tuxedocomputers.com/en
+
+Some traditional computer manufacturers now also sell laptops with Linux preinstalled:
+
+- Dell https://www.dell.com/en-us/work/shop/overview/cp/linuxsystems
+- Lenovo https://www.lenovo.com/us/en/d/linux-laptops-desktops/
+
+If those manufacturers do not ship the specific model preinstalled with Linux in your country. The fact that a model can be shipped with Linux is still a valuable information in terms of compatibility. 
+
+### Install Linux on an existing computer
+
+#### Hardware compatibility
+
+In general, most Intel/AMD based computers will support Linux pretty well nowadays.
+
+You can probably find reports on most computer models in the Linux Hardware project database https://linux-hardware.org/?view=computers
+
+Note that Linux compatibility can vary for special hardware like docking stations.
+
+#### Linux Distributions for beginners
+
+If you are new to Linux you will most likely start with one of the following distributions as they are very easy to install & use:
+
+- Ubuntu Desktop 
+  - Download: https://ubuntu.com/download/desktop 
+  - Installation walkthrough https://ubuntu.com/tutorials/install-ubuntu-desktop
+- Fedora Workstation 
+  - Download https://getfedora.org/en/workstation/download/
+
+The installation process should not be more difficult than installing Apple macOS or MS Windows. Most choices to make will be about the language, locals, timezone, keyboard & target installation disk.
+The copy on the disk is usually very fast nowadays (~10 20 minutes)  
+
+Note that you can still keep your previous operating system (like MS Windows) intact on a distinct partition and boot on it if necessary (like to perform firmware upgrades).
+
+**Attention**
+You should be careful to make sure full disk encryption is enabled during the installation:
+
+- For Fedora: Installation Destination screen > Encryption > ☑ Encrypt my data
+![](https://docs.fedoraproject.org/en-US/fedora/f34/install-guide/_images/anaconda/StorageSpoke.png)
+- For Ubuntu: Installation Type screen > Advanced features > ☑ Encrypt the new Ubuntu installation for security
+![](https://ubuntucommunity.s3-us-east-2.amazonaws.com/original/2X/4/46e9947d7b4ad3e96487d81cf61c327485d56b5a.png)
+
+#### Other Linux Distributions 
+
+Other popular Linux distributions includes:
+- Debian https://debian.org/
```

Comment:
I quite like PopOS for newb folks. default disk encryption, user friendly and clean interface, quality of life features with laptops and hybrid graphics drivers, based on Ubuntu but additional security checking for packages.

Reply:
Sure of course